CONTRIBUTING.md

```diff
@@ -49,5 +49,16 @@ For JS code, We have a python script to run all js tests.
 
 * `python tools/run-all-js-tests.py`
 
-## Recommended Chrome extensions
-* Hides generated (Go) files from GitHub pull requests: [GitHub PR gen hider](https://chrome.google.com/webstore/detail/mhemmopgidccpkibohejfhlbkggdcmhf).
+Jenkins automatically runs the Go and JS tests on every PR.
+
+### Perf tests
+
+By default, neither `go test` nor Jenkins run the perf tests, because they take a while.
+
+To run the tests yourself, use the `-perf` and `-v` flag to `go test`, e.g.:
+
+* `go test -v ./samples/go/csv/... -perf mem`
+
+See https://godoc.org/github.com/attic-labs/noms/go/perf/suite for full documentation and flags.
+
+To ask Jenkins to run the perf tests for you, reply (no quotes) "Perf this" to your PR. Yes, really.
```

Comment:
Can you add a note about ability to use "test this" for getting Jenkins to rebuild the PR too?